crates/bevy_math/src/aspect_ratio.rs

```diff
@@ -11,17 +11,60 @@ use bevy_reflect::Reflect;
 pub struct AspectRatio(f32);
 
 impl AspectRatio {
-    /// Create a new `AspectRatio` from a given `width` and `height`.
+    /// Standard 16:9 aspect ratio
+    pub const SIXTEEN_NINE: Self = Self(16.0 / 9.0);
+    /// Standard 4:3 aspect ratio
+    pub const FOUR_THREE: Self = Self(4.0 / 3.0);
+    /// Standard 21:9 ultrawide aspect ratio
+    pub const ULTRAWIDE: Self = Self(21.0 / 9.0);
+
+    /// Create a new [`AspectRatio`] from a given width and height.
     #[inline]
     pub fn new(width: f32, height: f32) -> Self {
```

Comment:
If we have `try_new`, then `new` should be the panicking counterpart.

Reply:
```rust
// new
Self::try_new(width, height).expect("invalid")

// try_new
match (width, height) {
    (w, h) if w == 0.0 || h == 0.0 => Err(Zero),
    (w, h) if w.is_infinite() || h.is_infinite() => Err(Infinite),
    (w, h) if w.is_nan() || h.is_nan() => Err(NaN),
    _ => Ok(Self(width / height))
}
```

Is this right?

Reply:
Yes, that's more or less it.